target/scripts/build/packages.sh

```diff
@@ -148,6 +148,14 @@ function _install_fail2ban
   rm fail2ban.deb fail2ban.deb.asc
 }
 
+function _install_getmail
+{
+  local GETMAIL_SID_DEB_URL='http://http.us.debian.org/debian/pool/main/g/getmail6/getmail6_6.18.9-1_all.deb'
+  curl -Lkso getmail6.deb  "${GETMAIL_SID_DEB_URL}"
+  dpkg -i getmail6.deb 2>&1
+  rm getmail6.deb
+}
```

Comment:
Can you please justify the reason for this over just installing the [Bullseye package](https://packages.debian.org/bullseye/getmail6)?

Reply:
6.14 was released in Jan 2021, nearly 2 years ago.  The main reason for adding getmail as an alternative for fetchmail is the ability to utilise oauth, 6.18 contains fixes for depricated Google flows and fixes Microsoft oauth issues.  Seemed like a good idea to use it.